src/components/ProgressIndicator/ProgressIndicatorDisplay.js

```diff
@@ -115,70 +115,72 @@ const ProgressIndicatorDisplay = ({
     defaultMessage: 'Progress step indicator icon',
   });
   const toggleStepsTitle = intl.formatMessage({
-    description: 'Active step toggle titles',
-    defaultMessage: 'Show/hide steps',
+    description: 'Active step title',
+    defaultMessage: 'Current step',
   });
 
   return (
     <Card blockClassName="progress-indicator" modifiers={expanded ? [] : ['mobile-collapsed']}>
-      {/* Turn a Div with an onClick event into a button for accesibility */}
-      <button
-        className={getBEMClassName('progress-indicator__mobile-header')}
-        aria-pressed={expanded ? 'true' : 'false'}
-        onClick={() => setExpanded(!expanded)}
-      >
-        <FAIcon
-          icon={expanded ? 'chevron-up' : 'chevron-down'}
-          modifiers={['normal']}
-          aria-label={ariaIconLabel}
-        />
-        <span
-          className={getBEMClassName('progress-indicator__active-step')}
-          aria-label={toggleStepsTitle}
+      <nav>
+        {/* Turn a Div with an onClick event into a button for accesibility */}
+        <button
+          className={getBEMClassName('progress-indicator__mobile-header')}
+          aria-pressed={expanded ? 'true' : 'false'}
+          onClick={() => setExpanded(!expanded)}
         >
-          {activeStepTitle}
-        </span>
-      </button>
+          <FAIcon
+            icon={expanded ? 'chevron-up' : 'chevron-down'}
+            modifiers={['normal']}
+            aria-label={ariaIconLabel}
+          />
+          <span
+            className={getBEMClassName('progress-indicator__active-step')}
+            aria-label={toggleStepsTitle + ': ' + activeStepTitle}
```

Comment:
this interpolation needs to be handled at the translation level rather than string concatenating - future languages may have a different sentence structure.

I'll pick this up myself to wrap up the PR.

Reply:
Thanks, I am very interested to see - could this be a right-to-left language issue?

Reply:
not necessarily, but there is potential that a language exists where a different structure is more natural.

Translators also have an option to then do something like `{activeStepTitle} (huidige stap)` instead of `Huidige stap: {activeStepTitle}`